app/templates/main/terms-of-use.html

```diff
@@ -1,18 +1,71 @@
+                </p>
+                <p class="govuk-body">
+                    Egestas maecenas pharetra convallis posuere morbi leo urna molestie. Dui faucibus in ornare quam viverra orci sagittis eu. Egestas erat imperdiet sed euismod nisi porta lorem mollis. Integer enim neque volutpat ac tincidunt. Vitae congue mauris rhoncus aenean. Tincidunt arcu non sodales neque sodales ut etiam. Blandit massa enim nec dui nunc mattis enim ut tellus. Sit amet mauris commodo quis imperdiet massa tincidunt nunc pulvinar. Euismod elementum nisi quis eleifend quam. Cursus sit amet dictum sit amet justo donec. Urna molestie at elementum eu facilisis sed odio.
+                </p>
+                <h2 class="govuk-heading-m">Anchor link four heading</h2>
+                <p class="govuk-body">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Pharetra convallis posuere morbi leo urna. Egestas tellus rutrum tellus pellentesque eu tincidunt. Euismod lacinia at quis risus sed vulputate odio ut enim. Feugiat nisl pretium fusce id velit. Enim sit amet venenatis urna cursus eget nunc.
+                </p>
+                <p class="govuk-body">
+                    Tincidunt ornare massa eget egestas purus viverra accumsan. Non arcu risus quis varius. Purus faucibus ornare suspendisse sed. Pellentesque adipiscing commodo elit at imperdiet dui accumsan sit. Elementum sagittis vitae et leo duis ut diam. Nulla at volutpat diam ut venenatis tellus in. Tempus imperdiet nulla malesuada pellentesque elit. Morbi blandit cursus risus at ultrices mi. Rhoncus dolor purus non enim praesent elementum facilisis leo vel. Maecenas volutpat blandit aliquam etiam. Nulla facilisi cras fermentum odio eu feugiat. Lorem sed risus ultricies tristique nulla aliquet enim.
+                </p>
+                <p class="govuk-body">
+                    Odio aenean sed adipiscing diam donec adipiscing tristique risus nec. Donec massa sapien faucibus et molestie ac. Faucibus ornare suspendisse sed nisi lacus sed viverra tellus in. A lacus vestibulum sed arcu non odio. Pharetra convallis posuere morbi leo urna molestie at elementum eu. Malesuada nunc vel risus commodo viverra maecenas. Sollicitudin nibh sit amet commodo nulla facilisi nullam. In egestas erat imperdiet sed euismod nisi porta lorem. Nisi porta lorem mollis aliquam ut porttitor leo a diam. Etiam non quam lacus suspendisse faucibus. Morbi tristique senectus et netus. Bibendum arcu vitae elementum curabitur vitae. Scelerisque eu ultrices vitae auctor eu augue ut lectus arcu. Lacus luctus accumsan tortor posuere ac ut consequat.
+                </p>
+                <div class="terms-of-use-back-to-top">
```

Comment:
this class name doesnt seem to be defined. Should you be using the __back-to-top style you created?